solutions/logging/client-experimentation/README.md

```diff
@@ -118,9 +128,9 @@ This package deploys the following resources:
             OR (LOG_ID("requests") AND resource.type="http_load_balancer")
         ```
 
-    - Excludes all Security logs: Cloud Audit, Access Transparency, and Data Access Logs
+  - Excludes all Security logs: Cloud Audit, Access Transparency, and Data Access Logs
 
-        ```yaml
```

Comment:
good catch on any code segments that have 4 spaces (would break the code flow api invoke)